src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -78,7 +78,7 @@ func (c *controller) WaitAndUpdateNodesStatus() {
 		time.Sleep(GeneralWaitInterval)
 		assistedInstallerNodesMap, err := c.ic.GetHosts(ignoreStatuses)
 		if err != nil {
-			c.log.WithError(err).Error("Failed to get node map from inventory")
+			c.log.WithError(utils.WrappedAssistedServiceErrorMessage(err, "")).Error("Failed to get node map from inventory")
```

Comment:
Consider moving the usage of this wrapper into ic ( I guess it's InventoryClient) this way it is used just in that module and the rest of the code won't be aware of the error behavior.

Reply:
+1